client/plots/singleCellPlot.js

```diff
@@ -84,6 +84,9 @@ class singleCellPlot {
 		})
 		this.tabs = []
 		const activeTab = state.config.activeTab
+		// shared isVisible function for tabs that require config.sample
+		const isVisible = () => this.state?.config.sample
```

Comment:
if the sample is preselected, like in the session, this would not capture it. That is why I check the init state

Reply:
if it's preselected, then it would be in the state as config.sample

Reply:
also, `showActiveTab()` is called in `main()`, so it would be checked/set against current state on every dispatch, not only on init

Reply:
but on init this.state is undefined, state uses the const state = this.getState(appState)

Reply:
It would check the active tab, not all the tabs @siosonel

Reply:
Please let me know if there tab visibility is not working as expected, otherwise not sure if what you are pointing out is relevant. `self.update()` in `toggeButtons.js` sets tab display in line 292, so it's checking more than which tab is active.
```js
if (tab.isVisible) tab.wrapper.style('display', tab => (config && tab.isVisible() ? '' : 'none'))
```